packages/react/toast/src/Toast.tsx

```diff
@@ -195,34 +204,63 @@ const ToastViewport = React.forwardRef<ToastViewportElement, ToastViewportProps>
       }
     }, [context.isClosePausedRef]);
 
+    const getSortedCandidates = React.useCallback(
+      ({ direction }: { direction: 'forwards' | 'backwards' }) => {
+        const items = getItems();
+        const candidateGroups = items.map((item) => {
+          const itemNode = item.ref.current!;
+          const itemGroup = [itemNode, ...getTabbableCandidates(itemNode)];
+          return direction === 'forwards' ? itemGroup.reverse() : itemGroup;
+        });
+        return (direction === 'forwards' ? candidateGroups : candidateGroups.reverse()).flat();
```

Comment:
Reading this confuses me a bit.
Why do we need to reverse the internal nodes when tabbing forward?
Once we're inside a toast, tabbing forward should be going natural order right?

Similarly, then the whole array is reverse when moving backwards this time, when to me it feels like it should be the opposite.

What am I not following here?